lib/phut/open_vswitch.rb

```diff
@@ -47,10 +58,10 @@ def options
          --pidfile=#{pid_file}
          --verbose=ANY:file:dbg
          --verbose=ANY:console:err
-         --log-file=#{Phut.settings['LOG_DIR']}/open_vswitch.#{@dpid}.log
+         --log-file=#{Phut.settings['LOG_DIR']}/open_vswitch.#{name}.log
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.